.github/workflows/spanner-load-tests.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Checks that are intended to run on PRs containing Java code.
+
+name: Spanner Load Tests
```

Comment:
Can you also remove the Spanner Load tests step for `spanner-pr.yml`?

Reply:
The load test step in spanner-pr and java-pr workflows have a condition ("Load test" comment) on which they get triggered. I thought they might be getting used during weekly release. @damccorm Can you please confirm?

If that is not used, I'll remove it.